src/Microsoft.Health.Dicom.Api/Features/Routing/KnownRoutes.cs

```diff
@@ -40,6 +41,9 @@ public static class KnownRoutes
         public const string ChangeFeed = "changefeed";
         public const string ChangeFeedLatest = ChangeFeed + "/" + "latest";
 
+        public const string CustomTagRoute = CustomTagsRouteSegment;
```

Comment:
rename as GetAllCustomTagsRoute  to keep consistency?

Reply:
This is the same route that's being used to add custom tags!